<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>701</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Newton Physics engine </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="versionLabel">
       <property name="text">
        <string>version:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayoutTop">
     <item row="2" column="0">
      <widget class="QLabel" name="frictionLabel">
       <property name="text">
        <string>mu</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="defaultButton">
       <property name="text">
        <string>default</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>apply</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="gravityValue"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="propertiesLabel">
       <property name="text">
        <string>Properties</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="frictionValue"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="gravityLabel">
       <property name="text">
        <string>gravity</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>210</y>
      <width>321</width>
      <height>253</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayoutBottom">
     <item row="0" column="0">
      <widget class="QLabel" name="objectLabel">
       <property name="text">
        <string>Object</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="setButton">
       <property name="text">
        <string>set</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="massLabel">
       <property name="text">
        <string>mass</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="restitutionLabel">
       <property name="text">
        <string>restitution</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QDoubleSpinBox" name="velocityValueY"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="shapeLabel">
       <property name="text">
        <string>shape</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="shapeComboBox">
       <item>
        <property name="text">
         <string>Circle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rectangle</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="massValue"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="velocityLabel">
       <property name="text">
        <string>velocity</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QDoubleSpinBox" name="sizeValueY"/>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="velocityValueX"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="sizeLabel">
       <property name="text">
        <string>size</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDoubleSpinBox" name="restitutionValue"/>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="sizeValueX"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="positionLabel">
       <property name="text">
        <string>position</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="positionValueX"/>
     </item>
     <item row="4" column="2">
      <widget class="QDoubleSpinBox" name="positionValueY"/>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="createButton">
       <property name="text">
        <string>create</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>540</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="leftWallCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>86</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="rightWallCheckBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>310</y>
      <width>86</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="roofCheckBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>86</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="groundCheckBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>580</y>
      <width>86</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>470</y>
      <width>107</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>814</x>
      <y>500</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>948</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
